ckanext/canada/schemas/presets.yaml

```diff
@@ -3142,10 +3148,20 @@ presets:
       mimetype: application/vnd.google-earth.kmz
     - value: LAS
     - value: LYR
+    - value: Leapfrog Mining
+      replaces: [leapfrog mining]
+    - value: Leapfrog Hydro
+      replaces: [leapfrog hydro]
+    - value: Leapfrog Viewer
+      replaces: [leapfrog viewer]
     - label: MapInfo
       value: TAB
+    - value: MapInfo Grid
```

Comment:
Might need to go back to the client and ask for the list of file extensions along with these format names